### PR TITLE
API-based WebApp

### PR DESCRIPTION
**Pull request by ismailakinkunmi**
- Display list of items on the home page
- Display number of likes.
- Add new like
- Add all items counter on the home page

**noted**
We started this project as two coding partners but due to personal responsibilities my partner could not continue so 
Anne-Lore De Vreese (Program Coach) said I should do my part and noted it in the PR also, Due to an access level I couldn't set Github pages for this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ cd into the folder:
 
 Open with any code editor
 
-install all the packages needed by this command :
+install all the packages needed with the command below :
```
Comment (moise10r) [inline README.md:30]: - [ ] I believe you have put a lot of work into this project :muscle: However your readme or PR doesn't contain a video of you explaining your project. It is a requirement to do a video presentation of your project to give visitors some insight into your project.
You can summarize your video using the following bullet points.

1. Recorded a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
4. The video contains a demo of the project, highlighting all the project features.
5. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

File: dist/main.js [hunk 2]
```diff
@@ -126,7 +126,7 @@ eval("__webpack_require__.r(__webpack_exports__);\n/* harmony import */ var _sty
...
+eval("__webpack_require__.r(__webpack_exports__);\n/* harmony export */ __webpack_require__.d(__webpack_exports__, {\n/* harmony export */   \"default\": () => (__WEBPACK_DEFAULT_EXPORT__)\n/* harmony export */ });\nconst getMovies = async () => {\r\n  const moviesData = await fetch(\"https://api.tvmaze.com/shows?page=2\");\r\n  const allData = await moviesData.json();\r\n\r\n  const data = allData.slice(0, 20).map((item) => item);\r\n  return data;\r\n};\r\n\r\n/* harmony default export */ const __WEBPACK_DEFAULT_EXPORT__ = (getMovies);\r\n\n\n//# sourceURL=webpack://capstone_project_2/./src/modules/apiCall.js?");
```
Comment (moise10r) [inline dist/main.js:129]: - [ ] There is an issue with your `comments` button, as required When a user clicks on the `comments` button, the Comments popup appears. and the web app is expected to retrieve data from the selected API and show details about the selected item. And likewise, retrieve data from the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) to show the item comments.
But it does not work as expected

**Review by moise10r — CHANGES_REQUESTED** [review]
### Hi **@TEAM,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
- You have followed  JavaScript best practices. ✔️ 
- Great unit tests Implementation ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ismailakinkunmi** [comment]
The comment button popup is not part of my task, kindly check the project
task in the kanban board also, I drop a note in the PR, and the REpo is not
mine the  [image: Screenshot (106).png]

File: dist/main.js (hunk 2 above)
Comment (ismailakinkunmi) [reply dist/main.js:129]: this comment button is for student B

**Review by EricMbouwe — APPROVED** [review]
### Hi @ismailakinkunmi, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/main.js (hunk 2 above)
Comment (moise10r) [reply dist/main.js:129]: Okay but I can the second reviewer approved it already

**Pull request closed without merging**